src/appshell/internal/applicationuiactions.cpp

```diff
@@ -152,6 +153,15 @@ const UiActionList ApplicationUiActions::m_actions = {
              Checkable::Yes
              ),
 
+    // Braille panel
+    UiAction(TOGGLE_BRAILLE_ACTION_CODE,
+             mu::context::UiCtxNotationOpened,
+             mu::context::CTX_ANY,
+             TranslatableString("action", "&Braille"),
```

Comment:
I changed this string from "Braille panel" to just "Braille" to match the other panels in the View menu.

For example, we have "Properties" in the View menu, not "Properties panel".

Reply:
Not sure about this part, because previous requirements need me to use "braille panel". No problem.